### ISSUE TITLE
Arrow and Pagination(Bullet) plugins have to click two times to move panel

### ISSUE THREAD

**Issue by hawkup** [naver/egjs-flicking-plugins#19]
## Description
I face the problem about Arrow and Pagination(Bullet) plugin. It has to click two times to move panel.

## Steps to check or reproduce
I created a sample project on Codesandbox. You could check on this [link](https://codesandbox.io/s/practical-monad-oqmc4?file=/src/components/Slide.vue). 

### Dependencies

```
@egjs/flicking-plugins: 4.1.0
@egjs/vue-flicking: 4.1.2
vue: 2.6.11
```

### Reproduce
Open the website from this project, and click to navigate panel by clicking on arrow or bullet. 
![Screen Shot 2564-07-19 at 22 32 30](https://user-images.githubusercontent.com/2748846/126187642-ef2d9b5c-7721-41eb-a006-8d36eb040d78.png)

**Comment by WoodNeck** [comment]
Hello! @hawkup!
Looks like `preventClickOnDrag` option is preventing the first click event triggered, it must be a bug.
Try to turn off that option for now, I'll fix that.

**Comment by WoodNeck** [comment]
We've released a new Flicking version(v4.2.1+) that fixes this, try that :)
I'll close this issue for now
